project/Build.scala

```diff
@@ -414,12 +414,6 @@ object DottyBuild extends Build {
         "org.scala-sbt" % "api" % sbtVersion.value % "test",
         "org.specs2" %% "specs2" % "2.3.11" % "test"
       ),
-      version := {
-        if (isNightly)
-          "0.1.1-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
-        else
-          "0.1.1-SNAPSHOT"
-      },
```

Comment:
When publishing nightlies we run `NIGHTLYBUILD=yes sbt ...` and thus L419 is used as the value of version. What was the reason for removing this?